doc/source/conduce-entities.rst

```diff
@@ -9,9 +9,22 @@
 Conduce Entities
 ================
 
-.. note:: WIP
 
-Conduce uses a concept called entity to define objects that exist in space and time.  An entity can exist for an instant or across a time span.  It can describe a point in space, a path (sequence of points), or a region (polygon).  An entity is defined by a series of samples that describe it's location in space over a period of time.  An entity also has attributes that describe it's characteristics. An entity's attributes can change over time.
+------------------
+What is an entity?
+------------------
+
+Conduce uses a concept called entity to define a person, place, or thing that exist in space and time.  An entity can exist for an instant or across a time span.  It can describe a point in space, a path (sequence of points), or a region (polygon).  An entity is defined by a series of samples that describe it's location in space over a period of time.  An entity also has attributes that describe it's characteristics. An entity's attributes can change over time.  Entities can be static or dynamic.  To understand the difference between the two, it is helpful to understand dynamic entities first.
+
+++++++++++++++++
+Dynamic entities
+++++++++++++++++
+Dynamic entities are composed of a sequence of samples.  The term sample is taken from the `signal processing domain <https://en.wikipedia.org/wiki/Sampling_(signal_processing)>`_ and describes the state of an entity at a discrete moment in time.  A sample describes an entity's location in space and other attributes.  These qualities can change from sample to sample so that when a visualization is constructed, these state changes can be visualized.  A sequence of samples that share the same ID define the entity, and so, an entity is a sequence of samples.  Entities defined this way are said to be `dynamic`.  Conduce does not require that an entity's sampling interval be regular.  This allows users to update entities based on events (state changes) rather than period (time changes).  However, it should be considered that some types of entities should be sampled regularly in order generate an accurate visual representations.
```

Comment:
As long as we're referencing signal processing, should we replace `dynamic` with `discrete`?

Reply:
The phrasing "it should be considered" sounds awkward to me.  Perhaps "it should be noted" instead?  Or perhaps make it active instead of passive: "However, note that some types..."

Reply:
any time you can replace four words with one, that's a win